CHANGELOG.md

```diff
@@ -1,5 +1,94 @@
 # Changelog
 
+## 2021.9.100 (6 October 2021)
+
+### Enhancements
+
+1. Split Notebook renderers into a separate extension ([Jupyter Notebook Renderers](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter-renderers)), allowing users to view Notebook outputs such as plotly, vega on [github.dev](https://github.dev).
+   ([#1909](https://github.com/Microsoft/vscode-jupyter/issues/1909))
+1. Added support to use Run by Line and Notebook Debugging in remote kernels.
+   ([#7576](https://github.com/Microsoft/vscode-jupyter/issues/7576))
+1. Added abiliyt to create notebooks using the menu options `File -> New File...`.
+   ([#7363](https://github.com/Microsoft/vscode-jupyter/issues/7363))
+1. Added a command to the command palette and an icon to the notebook toolbar to open the the table of contents for Notebooks.
+   ([#7305](https://github.com/microsoft/vscode-jupyter/issues/7305))
```

Comment:
@rchiodo @IanMatthewHuff @DavidKutu Please review the news (this is now in the release branch)